src/python/pants/backend/terraform/dependency_inference.py

```diff
@@ -104,6 +105,7 @@ async def setup_process_for_parse_terraform_module_sources(
             argv=request.paths,
             input_digest=request.sources_digest,
             description="Parse Terraform module sources.",
```

Comment:
Could we also make this message dynamic? Say that folder that is being parsed?

Reply:
Sure.